data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,59 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Saryd"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "saryd" "folklore"
+	on offer
+		conversation
+			`Walking through the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`
+			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" He asks.`
```

Comment:
```suggestion
			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" he asks.`
```
Here's a quick rule with dialogue, if you're interested: if you're finishing with a question/exclamation mark (like you've done here), only capitalize the next word if it starts a new sentence. In this case, `he says` is finishing the sentence which includes the dialogue before it. It's a little bit of a strange case.
I have this strange feeling that I saw this somewhere else recently and didn't correct it. It's probably not worth digging for, but if anything comes to mind, let me know.